### PR TITLE
Add resources for w1s3

### LINKED ISSUE #442: [Content Bug] Response of Week 1 Step 3
[https://github.com/bitprj/Intro-To-Serverless/blob/main/.bit/responses/1.3-Week%201%20Step%203.md](https://github.com/bitprj/Intro-To-Serverless/blob/main/.bit/responses/1.3-Week%201%20Step%203.md)

**Identify Steps:**
* Step  3

**Week:**

week 1

## Describe the Issue

I think it might be very helpful for students to have more references concerning the HTTP methods. I would therefore recommend to add the reference to the Mozilla Developer documentation.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods

Especially for the `PUT` an example might be helpful. You could reference [https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PUT](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PUT).

As you also reference REST as an additional information it might be interesting for some students where this architectural pattern comes from:

- [Overall PhD Thesis](https://www.ics.uci.edu/~fielding/pubs/dissertation/top.htm) 
- [Section on REST](https://www.ics.uci.edu/~fielding/pubs/dissertation/rest_arch_style.htm)

## Proposed Fix

- Add references for HTTP methods
- Add reference to "REST PhD Thesis"

### PR DESCRIPTION
**Pull request by emsesc**
## Changes made
* Added REST resource
* Added HTTP methods resource

Closes #442

### CODE REVIEW

File: .bit/responses/1.3-Week 1 Step 3.md [hunk 1]
```diff
@@ -43,15 +43,15 @@ Clients communicate with servers with through `HTTP requests`. For example, when
 <summary>❓ What are some examples of request types??</summary>
```
Comment (ganning127) [inline .bit/responses/1.3-Week 1 Step 3.md:43]: Maybe make it to just one question mark at the end so it's less aggressive haha

**Review by ganning127 — CHANGES_REQUESTED** [review]
Lokve

File: .bit/responses/1.3-Week 1 Step 3.md (hunk 1 above)
Comment (lechnerc77) [reply .bit/responses/1.3-Week 1 Step 3.md:43]: +1

**Review by lechnerc77 — APPROVED** [review]
LGTM

**Comment by lechnerc77** [comment]
As @ganning127 has requested one change, the rest is fine, so I approved it

**Review by ganning127 — APPROVED** [review]

**Pull request merged**